easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033-fosscuda-2019b-Java-1.8.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'rstudio'
+version = '1.2.5033'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://www.rstudio.com/'
+description = """This RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:  rserver --server-daemonize=0 --www-port 8787 --rsession-which-r=$(which R) --auth-none=1
```

Comment:
Is there a specific reason why `--auth-none=1` is being suggested?

Reply:
This was suggested by a user. I think I'll remove it for the official EC

Reply:
> Is there a specific reason why `--auth-none=1` is being suggested?

The reason is that the open-source version of RStudio Server does not support any other authentification method (such as e.g. LDAP, which is currently used on our system) except the system users, created with `useradd` .

Reply:
@lazariv Ah, that makes sense, but how does RStudio auth actually resolve system users? Do they need to be in /etc/passwd, or does it work with sss_cache and friends that then get their data from LDAP? Those users are able to log into the machine.

Reply:
@terjekv As far as I know, it uses PAM for authentification.  Some more information is available here [https://jstaf.github.io/2018/06/20/rstudio-server-semi-pro.html](url)

Reply:
https://jstaf.github.io/2018/06/20/rstudio-server-semi-pro.html was quite helpful, thanks! Although I'm not sure I want to work on PAM with something coming from modules. :-)